include/minit-assets.php

```diff
@@ -230,6 +230,17 @@ protected function get_asset_relative_path( $handle ) {
 			return false;
 		}
 
+		$local_path = get_local_path_from_url( $item_url );
```

Comment:
@szepeviktor We should call the method of this class, correct? `$this->get_local_path_from_url()`.